src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbBlockListPropertyEditor.component.js

```diff
@@ -17,18 +17,18 @@
             controller: BlockListController,
             controllerAs: "vm",
             bindings: {
-                model: "=",
-                propertyForm: "="
+                model: "="
             },
             require: {
+                propertyForm: "^form",
```

Comment:
Happy that you know this is right for all property-editors, I guess I didn't have that big picture :-)